lib/collection/src/operations/snapshot_ops.rs

```diff
@@ -16,6 +16,7 @@ use crate::operations::types::CollectionResult;
 #[derive(Debug, Deserialize, Serialize, JsonSchema, Default, Clone, Copy)]
 #[serde(rename_all = "snake_case")]
 pub enum SnapshotPriority {
+    LocalOnly,
```

Comment:
Maybe just `Local`? Or something like `Disabled`?

Reply:
I don't like `LocalOnly` much, but I feel like `Local` or `Disabled` would be even worse. :/

Reply:
@agourlay @timvisee Do you guys have a better suggestion, maybe?

`SnapshotPriority` controls how the cluster is synchronized after restoring collection/shard snapshot.

- `Snapshot` means "overwrite whatever is in the cluster with restored snapshot data"
  - and so, after snapshot is recovered, all *other* replicas of restored collection/shard are marked as `Dead`
  - and shard transfer replicates recovered data on other shards
- `Replica` means "overwrite restored data with data in the cluster"
  - and so, after the snapshot is recovered, *the recovered* replica is marked as `Dead`
  - and shard transfer overwrites recovered data (fixing any possible inconcistencies)

We want to add another priority, that would simply restore the collection/shard, *activate it*... and do nothing else. No further synchronization with the cluster.

Reply:
So we have an enum. specifying if we either push or pull updates following a restore.

What about `NoSync`?